src/ts/background/types.ts

```diff
@@ -36,15 +38,7 @@ export type MessageExtrinsicSignCancel = {
   id: string
 }
 
-export type MessageExtrinsicSign = {
-  address: string,
-  blockHash: string,
-  blockNumber: number,
-  era?: string,
-  genesisHash: string,
-  method: string,
-  nonce: string
-}
+export type MessageExtrinsicSign = SignerPayload
```

Comment:
should we just use type `SignerPayload ` directly?

Reply:
This code is copied from Jaco's. I think he keeps the `MessageExtrinsicSign` is to minimise changes, since there are `MessageExtrinsicSignApprove`, `MessageExtrinsicSignCancel` etc. The interface is probably not stable yet. I keep it the same as Jaco's to make it easier for future's sync